### ISSUE TITLE
Using --cert-forward option results in a crash using the OS X binary install

### ISSUE THREAD

**Issue by iragsdale** [mitmproxy/mitmproxy#471]
```
Traceback (most recent call last):
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/libmproxy.proxy.server", line 99, in handle
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/libmproxy.protocol.http", line 982, in handle_messages
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/libmproxy.protocol.http", line 1055, in handle_flow
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/libmproxy.protocol.http", line 1248, in process_request
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/libmproxy.protocol.http", line 1395, in process_connect_request
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/libmproxy.proxy.server", line 206, in establish_ssl
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/libmproxy.proxy.server", line 257, in find_cert
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/netlib.certutils", line 270, in gen_pkey
  File "/Users/cortesi/tmp/pyinstaller/PyInstaller/loader/pyi_importers.py", line 276, in load_module
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/netlib.certffi", line 20, in <module>
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/cffi.api", line 340, in verify
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/cffi.verifier", line 73, in load_library
  File "/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/cffi.verifier", line 125, in _write_source
IOError: [Errno 2] No such file or directory: '/Users/cortesi/mitmproxy/mitmproxy/build/mitmproxy/out00-PYZ.pyz/__pycache__/_cffi__x88a8fd90x94e38fe2.c'

mitmproxy has crashed!
Please lodge a bug report at: https://github.com/mitmproxy/mitmproxy
```

**Comment by mhils** [comment]
Thanks for the report!

**Comment by iragsdale** [comment]
Sure thing! Installing from source appears to work OK.

**Comment by gtklocker** [comment]
I'm facing the same issue on mitmproxy installed via pip.

```
mitmproxy has crashed!
Please lodge a bug report at: https://github.com/mitmproxy/mitmproxy
127.0.0.1:55860: Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/libmproxy/proxy/server.py", line 92, in handle
    self.establish_ssl(client=client_ssl, server=server_ssl)
  File "/Library/Python/2.7/site-packages/libmproxy/proxy/server.py", line 206, in establish_ssl
    cert, key, chain_file = self.find_cert()
  File "/Library/Python/2.7/site-packages/libmproxy/proxy/server.py", line 257, in find_cert
    return self.server_conn.cert, self.config.certstore.gen_pkey(self.server_conn.cert), None
  File "/Library/Python/2.7/site-packages/netlib/certutils.py", line 270, in gen_pkey
    from . import certffi
  File "/Library/Python/2.7/site-packages/netlib/certffi.py", line 20, in <module>
    extra_compile_args=['-w']
  File "/Library/Python/2.7/site-packages/cffi/api.py", line 367, in verify
    lib = self.verifier.load_library()
  File "/Library/Python/2.7/site-packages/cffi/verifier.py", line 95, in load_library
    self._write_source()
  File "/Library/Python/2.7/site-packages/cffi/verifier.py", line 183, in _write_source
    with open(self.sourcefilename, "w") as fp:
IOError: [Errno 2] No such file or directory: '/Library/Python/2.7/site-packages/netlib/__pycache__/_cffi__x88a8fd90x94e38fe2.c'
```

```
$ pip show mitmproxy
---
Name: mitmproxy
Version: 0.11.3
Location: /Library/Python/2.7/site-packages
Requires: urwid, pyasn1, tornado, lxml, pyOpenSSL, Pillow, configargparse, netlib
```

**Comment by mhils** [comment]
@gtklocker, your issue is probably a duplicate of https://github.com/mitmproxy/netlib/issues/32 - could you verify that the workaround posted there works for you?
